Libraries/Microsoft.Teams.Apps/Activities/Invokes/MessageExtensions/SubmitActionActivity.cs

```diff
@@ -15,7 +15,7 @@ public class SubmitActionAttribute() : InvokeAttribute(Api.Activities.Invokes.Na
 
 public static partial class AppInvokeActivityExtensions
 {
-    public static App OnSubmitAction(this App app, Func<IContext<MessageExtensions.SubmitActionActivity>, Task<object?>> handler)
+    public static App OnMessageExtensionSubmitAction(this App app, Func<IContext<MessageExtensions.SubmitActionActivity>, Task<object?>> handler)
```

Comment:
Hi Alex, I needed to make this change since there is another OnSubmitAction extention method on app, pls let me know if you think there might be other way to handle this.

Reply:
thats fine, but if we are going to rename one we need to rename them all to stay consistent